packages/native/types/tsconfig.json

```diff
@@ -6,6 +6,10 @@
     "lib": ["es6", "dom"],
     "module": "commonjs",
     "noEmit": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "strictNullChecks": true,
+    "strictFunctionTypes": true,
```

Comment:
all of those should already be enabled by the `strict: true` that we already have here, was this change required for anything?
```suggestion
```